src/pilet-debug.test.ts

```diff
@@ -40,4 +41,200 @@ runTests('pilet-debug', ({ test, setup }) => {
+        integrity: expect.anything(),
+      });
+
+      await expect(page).toHaveSelectorCount('.pi-tile', 1);
+
+      await expect(page).toMatchText('.pi-tile', 'Welcome to Piral!');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-default-schema-v2',
+    'can produce a debug build with default schema v2',
+    ['debug.pilet'],
+    async (ctx) => {
+      const port = await getFreePort(1256);
+      const cp = ctx.runAsync(`npx pilet debug --port ${port} --schema`);
+
+      await cp.waitUntil('Ready', 'The bundling process failed');
+
+      await page.goto(`http://localhost:${port}`);
+
+      const res = await axios.get(`http://localhost:${port}/$pilet-api`);
+      const pilets = res.data;
+
+      expect(pilets).toEqual({
+        name: expect.anything(),
+        version: expect.anything(),
+        link: expect.anything(),
+        spec: 'v2',
+        requireRef: expect.anything(),
+        dependencies: expect.anything(),
+      });
+
+      await expect(page).toHaveSelectorCount('.pi-tile', 1);
+
+      await expect(page).toMatchText('.pi-tile', 'Welcome to Piral!');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-none-schema',
+    'can produce a debug build with none schema',
+    ['debug.pilet'],
+    async (ctx) => {
+      const port = await getFreePort(1256);
+      const cp = ctx.runAsync(`npx pilet debug --port ${port} --schema none`);
+
+      await cp.waitUntil('Ready', 'The bundling process failed');
+
+      await page.goto(`http://localhost:${port}`);
+
+      const res = await axios.get(`http://localhost:${port}/$pilet-api`);
+      const pilets = res.data;
+
+      expect(pilets).toEqual({
+        name: expect.anything(),
+        version: expect.anything(),
+        link: expect.anything(),
+        spec: 'v0',
+        hash: expect.anything(),
+        noCache: expect.anything(),
+      });
+
+      await expect(page).not.toHaveSelectorCount('.pi-tile', 1);
+
+      await expect(page).not.toMatchText('.pi-tile', 'Welcome to Piral!');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-an-external-feed',
+    'can produce a debug build with an external feed',
+    ['debug.pilet'],
+    async (ctx) => {
+      const port = await getFreePort(1256);
+      const cp = ctx.runAsync(`npx pilet debug --port ${port} --feed "https://feed.piral.cloud/api/v1/pilet/sample"`);
+
+      await cp.waitUntil('Ready', 'The bundling process failed');
+
+      await page.goto(`http://localhost:${port}`);
+
+      await expect(page).toHaveSelectorCount('.mario-tile', 1);
+      await expect(page).toHaveSelectorCount('.spaceshoot-tile', 1);
+      await expect(page).toHaveSelectorCount('.pi-tile', 1);
+      await expect(page).toMatchText('.pi-tile', 'Welcome to Piral!');
+      await expect(page).toMatchText('.mario-tile', 'Mario5');
+      await expect(page).toMatchText('.spaceshoot-tile', 'Spaceshoot');
+    },
+  );
+
+  test(
+    'debug-standard-template-with-an-external-feed',
```

Comment:
Hm that is just a copy of the test above. What's the reason for having this duplicate?